### ISSUE TITLE
Jetty 9.4 support

### ISSUE THREAD

**Issue by sparhomenko** [akhikhl/gretty#365]
Our team is using Gretty for the integration testing and in the local development environment for an application that is deployed on Google App Engine Flexible Environment. We would like to be able to use the same Jetty version on all environments. Unfortunately, this does not seem to be possible as GAE Flexible is running Jetty 9.4, and if we configure Gretty to use the same:
```groovy
configurations.grettyRunnerJetty93.resolutionStrategy.eachDependency {
    if (requested.group.startsWith('org.eclipse.jetty') && requested.group != 'org.eclipse.jetty.toolchain') {
        useVersion '9.4.5.v20170502' // latest
    }
}
```
Gretty startup fails with `java.lang.ClassNotFoundException: org.eclipse.jetty.server.session.HashSessionManager`. Jetty 9.4 upgrade doc indeed [mentions](https://www.eclipse.org/jetty/documentation/9.4.x/upgrading-jetty.html#_session_management) some changes around session management, so some changes are likely to be required in Gretty code base. Jetty 9.4 usage [is growing](https://w3techs.com/technologies/history_details/ws-jetty/9) and by the end of the year it is likely to be more popular than 9.3 between Java8+ teams, so adjusting Gretty to support it might be worthwhile.

**Comment by alexolivas** [comment]
+1

**Comment by akhikhl** [comment]
The issue will be resolved in Gretty 2.0.0, which I plan to release today or tomorrow.

**Comment by sparhomenko** [comment]
@akhikhl We tried 9.4 support in 2.0.0 and it works well, thank you.
